### PR TITLE
"Shop" => "Shop (Unspecified type)" for the searchable shop=yes preset

### PR DESCRIPTION
**Pull request by Dimitar5555**
### Description, Motivation & Context

This PR changes the name of the searchable `shop=yes` preset to "Shop (Unspecified type)" (was: "Shop").

### Related issues

https://github.com/openstreetmap/id-tagging-schema/issues/1414. 

This PR addresses part of the issue: 
> When you search for "shop" **first match is shop=yes without even warning that it is an unspecific value**, to be avoided
### Links and data

**Relevant OSM Wiki links:**
N/A

**Relevant tag usage stats:**
N/A

### Checklist and Test-Documentation Template

<details><summary>Read on to get your PR merged faster…</summary>

Follow these steps to test your PR yourself and make it a lot easier and faster for maintainers to check and approve it.

**This is how it works:**
1. After you submit your PR, the system will create a preview and comment on your PR:
   > 🍱 You can preview the tagging presets of this pull request here.
   If this is your first contribution to this project, the preview will not happen right away but requires a click from one of the project members. We will do this ASAP.

2. Once the preview is ready, use it to test your changes.

3. Now copy the snippet below into a new comment and fill out the blanks.

4. Now your PR is ready to be reviewed.

```
## Test-Documentation

### Preview links & Sidebar Screenshots

<!-- Use the preview to find examples, select the feature in question and **copy this link here**.
     Find examples of nodes/areas. Find examples with a lot of tags or very few tags. – Whatever helps to test this thoroughly.
     Add relevant **screenshots** of the sidebar of those examples. -->

<!-- FYI: What we will check:
     - Is the [icon](https://github.com/ideditor/schema-builder/blob/main/ICONS.md) well chosen.
     - Are the fields well-structured and have good labels.
     - Do the dropdowns (etc.) work well and show helpful data. -->

### Search

<!-- **Test the search** of your preset and share relevant **screenshots** here.
     - Test the preset name as search terms.
     - Also test the preset terms and aliases as search terms (if present). -->

### Info-`i`

<!-- **Test the info-i** for your fields and preset and share relevant **screenshots** here.
     The info needs to help mappers understand the preset and when to use it.
     [Learn more…](https://github.com/openstreetmap/id-tagging-schema/blob/main/CONTRIBUTING.md#info-i)
 -->

### Wording

- [ ] American English
- [ ] `name`, `aliases` (if present) use Title Case
- [ ] `terms` (if present) use lower case, sorted A-Z
<!-- Learn more in https://github.com/openstreetmap/id-tagging-schema/blob/main/GUIDELINES.md#2-design-the-preset -->
```

</details>

### CODE REVIEW

**Comment by Dimitar5555** [comment]
The other preset is in `presets/shop/_yes.json`. I'm not sure why it doesn't find it.

File: data/presets/shop.json [hunk 1]
```diff
@@ -38,6 +38,6 @@
         "retailer",
         "store"
     ],
-    "name": "Shop",
+    "name": "{shop/_yes}",
```
Comment (tordans) [inline data/presets/shop.json:41]:
```suggestion
    "name": "{shop/yes}",
```

When referencing a name, you have to omit the underscore. This is explained in https://github.com/ideditor/schema-builder/pull/148/files but not merged yet.

**Comment by tordans** [comment]
This will now take the existing name from https://github.com/openstreetmap/id-tagging-schema/blob/main/data/presets/shop/_yes.json

It looks like this:
Bildschirmfoto 2024-12-25 um 20 34 34
Bildschirmfoto 2024-12-25 um 20 34 55


https://pr-1415--ideditor-presets-preview.netlify.app/id/dist/#background=Bing&disable_features=boundaries&locale=en&map=20.00/52.47409/13.44561

**Review by tordans — APPROVED** [review]

**Comment by tordans** [comment]
Please read https://github.com/openstreetmap/id-tagging-schema/issues/1414#issuecomment-2561986215 for some additional thought if we actually want this change that I just merged.

**Comment by Dimitar5555** [comment]
I didn't expect such consequences... I'm not sure if it's desired or not.

image

**Comment by tordans** [comment]
@Dimitar5555 thanks for finding this, that was the case that I was looking for but did not manage to find when testing it yesterday.

I will revert the change now so we can thin, about this again…

**Review by Starmobilerepair — COMMENTED** [review]
(Deleted)

**Comment by mnalis** [comment]
As a low-tech solution:
Perhaps `Shop (Unknown type)` would be better than `Shop (Unspecified type)`? 

- it would be still correct for `shop=yes` generic shop type (as specific type of that shop **is** _unknown_), and
- It would also not lie when `shop=some_random_thing` was added -- it would still provide truthful feedback to the user (yeah, that shop _has_ specific value, but that _specific value_ ***is unknown***).

**Pull request merged**